libraries/abstractions/ble_hal/test/src/iot_test_ble_hal_common.c

```diff
@@ -724,22 +725,25 @@ static void prvCreateCharacteristic( BTService_t * xSrvc,
     TEST_ASSERT_EQUAL( eBTStatusSuccess, xStatus );
     CharAddedComplete = true;
```

Comment:
Nit: Not this PR. but variable name is inconsistent.

Reply:
Changed it to bCharAddedComplete